src/test/java/redis/clients/jedis/modules/search/AggregationBuilderTest.java

```diff
@@ -83,6 +84,17 @@ public void testAggregations() {
     assertNotNull(r2);
     assertEquals("abc", r2.getString("name"));
     assertEquals(10, r2.getLong("sum"));
+
+    r = new AggregationBuilder()
+        .groupBy("@name", Reducers.sum("@count").as("sum"))
+        .verbatim()
+        .timeout(1000)
+        .params(new Params("@name", "abc"))
+        .dialect(2);
+
+    // actual search
+    res = client.ftAggregate(index, r);
+    assertEquals(2, res.totalResults);
```

Comment:
It'd be nicer to actually test the effects of the arguments.
Also, can be considered a separate test method.

Reply:
Wrote individual tests for each arguments to test their effects.